src/blog/dark-mode-with-style-dictionary/index.mdx

```diff
@@ -0,0 +1,1352 @@
+
+```css
+@media (prefers-color-scheme: dark) {
+  /* Dark mode styles here */
+}
+
+@media (prefers-color-scheme: light) {
+  /* Light mode styles here */
+}
+```
+
+You can conditionally load CSS files based on media query:
+
+```html
+<!-- No media query, will always be loaded -->
+<link rel="stylesheet" href="/base.css" />
+<!-- Will only be loaded if user prefers light color scheme or has no preference -->
+<link rel="stylesheet" href="/light.css" media="(prefers-color-scheme: light) or (prefers-color-scheme: no-preference)" />
+<!-- Will only be loaded if the user prefers a dark color scheme -->
+<link rel="stylesheet" href="/dark.css" media="(prefers-color-scheme: dark)" />
+```
+
+You can use the media query to show different images:
+
+```html
+<picture>
+  <!-- Use this image if the user's OS setting is light or unset -->
+  <source srcset="photo-light.png" media="(prefers-color-scheme: light) or (prefers-color-scheme: no-preference)">
+  <!-- Use this image if the user's OS setting is dark -->
+  <source srcset="photo-dark.png" media="(prefers-color-scheme: dark)">
+  <img src="photo-light.png" />
+</picture>
+```
+
+You can use the media query in Javascript too:
+
+```javascript
+if (window.matchMedia && window.matchMedia(`(prefers-color-scheme: dark)`).matches) {
+  // dark mode
+} else {
+  // light mode
+}
+
+// You can add an event listener to do something when the media query changes
+// because a user changed their preference
+var darkQuery = window.matchMedia('(prefers-color-scheme: dark)');
+darkQuery.addListener(function(e) {
+  var newTheme = e.matches ? 'dark' : 'light';
+  // do something here
+});
+```
+
+If you wanted, you could write completely separate sets of styles for light and dark mode for each component. But that sounds a bit tedious and error prone. Instead, people tend to use [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) (sometimes called CSS variables) in addition to the `prefers-color-scheme` media query:
+
+```css
+:root {
+  --color-background-primary: #fff;
+  --color-font-primary: #000;
+}
+
+.card {
+  background-color: var(--color-background-primary);
+  color: var(--color-font-primary);
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --color-background-primary: #000;
+    --color-font-primary: #fff;
+  }
+}
+```
+
+Using CSS custom properties and having a layer of semantic tokens allows you to not have to re-style every component for light and dark mode. The above example is very simple, but can be expanded to a fit the needs of a full design system. Also if you are using inline SVG (as opposed to using SVG in an `<img>` tag), your SVG can be styled with CSS and can use CSS custom properties as well! One more quick tip before we move on: you can use the `currentColor` property in SVG as well to have the graphic inherit its color from its parent. This is useful if you have an icon inside a link or button for example. 
```

Comment:
Also if you are using inline SVG (as opposed to using SVG in an `<img>` tag), your SVG can be styled with CSS and can use CSS custom properties as well!

maybe better:
If you are using inline SVG (as opposed to using SVG in an <img> tag), then your SVG can be styled with CSS and also use CSS custom properties!